### ISSUE TITLE
【React SSR】When I used the latest version, I reported an error in chrome 84, But I went back to **v5.12.0** and I didn't have this problem。

### ISSUE THREAD

**Issue by zhangKunUserGit** [gregberge/loadable-components#618]
## 💬 Questions and Help
React SSR
When I used the latest version, I reported an error in chrome 84
![image](https://user-images.githubusercontent.com/13297317/90745130-c1422a00-e302-11ea-9d05-f40d29344885.png)
But I went back to **v5.12.0** and I didn't have this problem。

It is found that the new version **v5.13.1** will confuse the preloaded CSS with JS, which is not arranged in the order of JS code execution
![image](https://user-images.githubusercontent.com/13297317/90747554-a623ea00-e303-11ea-9b91-785b53bb6af1.png)

The v5.12.0 version is in order
![image](https://user-images.githubusercontent.com/13297317/90747950-21859b80-e304-11ea-8163-9452b5f97d24.png)

**Comment by theKashey** [comment]
Strange, 5.13 is fixing exactly this issue in 5.12

**Comment by zhangKunUserGit** [comment]
> Strange, 5.13 is fixing exactly this issue in 5.12

I couldn’t understand it at that time. It was okay before (the page flashed suddenly and the screen went blank). It’s been a long time since the version was released. There were no major changes to the code. I just modified some of the page styles, compared and packaged all dependencies, and found that only this version is not The fixed version number uses `^5.12.0`, and node_modules is installed with v5.13.1. After the fixed version, there is no such problem. Comparing the html after packaging and running, it is found that the order of css and js is messy.
